README.md

```diff
@@ -63,6 +63,33 @@ adaptor = StanHMCAdaptor(MassMatrixAdaptor(metric), StepSizeAdaptor(0.8, integra
 samples, stats = sample(hamiltonian, proposal, initial_θ, n_samples, adaptor, n_adapts; progress=true)
 ```
 
+### Parallel sampling 
+
+AdvancedHMC enables parallel sampling (either distributed or multi-thread) via Julia's [parallel computing functions](https://docs.julialang.org/en/v1/manual/parallel-computing/).
+It also supports vectorized sampling for static HMC and has been discussed in more detail in the documentation [here](https://turing.ml/dev/docs/library/advancedhmc/parallel_sampling).
+
+The below example utilizes the `@threads` macro to sample 4 chains across 4 threads.
+
+```julia
+#Ensure that julia was launched with appropriate number of threads
+println(Threads.nthreads())
+
+#Number of chains to sample
+nchains = 4
+
+#Cache to store the chains
+chains = Vector{Any}(undef, nchains)
+
+#Adjust the `verbose` flag as per need
+Threads.@threads for i in 1:nchains
+  chains[i] = sample(hamiltonian, proposal, initial_θ, n_samples, adaptor, n_adapts; verbose = false)
+end
+
+#Each element of the `chains` vector contains a pair of `samples` and `stats` 
+#Concatenate the chains into a single chain
+aggregate_chain vcat([chain[1] for chain in chains]...)
```

Comment:
So this merges 4 chains of length N into a chain of length 4N, right? I'm not we want to do this here, instead, we can update the in-loop statement to
```julia
samples, stats = 
chains[i] = samples
```
And how to use the four chains is up-to users (either concatenated or treating them as 4 independent chains).

BTW you missed a `=` here.